build.sbt

```diff
@@ -14,12 +14,15 @@ val akkaPersistenceInMemVersion     = "2.5.1.1"
 val akkaPersistenceCassandraVersion = "0.55"
 val akkaHttpVersion                 = "10.0.10"
 val akkaHttpCirceVersion            = "1.18.0"
+val elasticSearchVersion            = "5.5.3"
```

Comment:
can we use version 6.1.1 ?

Reply:
[org.elasticsearch.client.rest](https://mvnrepository.com/artifact/org.elasticsearch.client/rest) has as latest the version 5.5.3

Reply:
http://search.maven.org/#artifactdetails%7Corg.elasticsearch.client%7Celasticsearch-rest-client%7C6.1.1%7Cjar